Hushes some (but not all warnings) relating to redefinitions of this define
--- a/vmci-only/shared/vm_device_version.h	2015-02-07 03:11:55.000000000 +0300
+++ c/vmci-only/shared/vm_device_version.h	2015-02-24 03:58:06.041605450 +0300
@@ -53,7 +53,9 @@
  *    VMware HD Audio codec
  *    VMware HD Audio controller
  */
+#ifndef PCI_VENDOR_ID_VMWARE
 #define PCI_VENDOR_ID_VMWARE                    0x15AD
+#endif
 #define PCI_DEVICE_ID_VMWARE_SVGA2              0x0405
 #define PCI_DEVICE_ID_VMWARE_SVGA               0x0710
 #define PCI_DEVICE_ID_VMWARE_VGA                0x0711
--- a/vmnet-only/vm_device_version.h	2015-02-07 03:54:16.000000000 +0300
+++ c/vmnet-only/vm_device_version.h	2015-02-24 03:58:06.044604981 +0300
@@ -53,7 +53,9 @@
  *    VMware HD Audio codec
  *    VMware HD Audio controller
  */
+#ifndef PCI_VENDOR_ID_VMWARE                    
 #define PCI_VENDOR_ID_VMWARE                    0x15AD
+#endif
 #define PCI_DEVICE_ID_VMWARE_SVGA2              0x0405
 #define PCI_DEVICE_ID_VMWARE_SVGA               0x0710
 #define PCI_DEVICE_ID_VMWARE_VGA                0x0711
